internal_docs/releasing/releasing.md

```diff
@@ -27,17 +27,13 @@ v8.0.0 // After implementing backwards incompatible changes
 
 (From [jvandemo.com](https://www.jvandemo.com/a-simple-guide-to-semantic-versioning/))
 
-3. Draft release notes in a Google Doc.
+3. Store the release notes (based on the notes in CHANGELOG.md) in a temporary text file (eg. in a Google Doc).
```

Comment:
I think the process that leads to the Google Doc is more involved, it's worth checking with @joelanman when he's back.

Reply:
yeh you're right. Just discussed this with @HannahJMWood  - we'll drop this as a step in the process and add it as a line before the steps:

Before the release, the content designer will draft the release notes, based on the changelog but written to be clear for our audience.

@oli-rose28 something like that?